### DISCUSSION TITLE
Why using nlohmann JSON library instead of QJsonDocument ?

### DISCUSSION THREAD

**Discussion by AlexandrePTJ** [KDAB/KDDockWidgets#463] [General]
I don't want to question your choice, but I would like to understand, for my personnal culture, the reasons for this technical choice.
Thanks you.

**Comment by iamsergio** [comment]
Hello Alexandre, 

KDDW layouting engine can be reused by non-Qt frameworks (currently Flutter and Slint).
It's easier for those frameworks to depend on a small library such as nlohmann, than to bring in the entire QtCore.

I don't have any technical problems with QJsonDocument otherwise.

**Reply by AlexandrePTJ** [reply]
Ok I understand. Thanks you for this explanation.
